.github/workflows/linux-32bit-build-and-test.yaml

```diff
@@ -51,6 +51,10 @@ jobs:
     - name: Checkout TimescaleDB
       uses: actions/checkout@v1
 
+    - name: Fetch all tags
+      # We need the tags to be able to build a downgrade script.
+      run: git fetch --tags && git tag
+
```

Comment:
i guess this is no longer needed with the changed default

Reply:
Removed.